src/docs/configuration.html

```diff
@@ -4,12 +4,29 @@
 
 <h1>Overlay Configuration API</h1>
 
-<p>The <dfn style="font-weight: bold;">overlay configuration API</dfn> is the set of query parameters which can be provided as part of your <b>showmy.chat</b> overlay URL to customize its behavior. Every query parameter is optional.</p>
+<p>
+	The <dfn style="font-weight: bold">overlay configuration API</dfn> is the set
+	of query parameters which can be provided as part of your
+	<b>showmy.chat</b> overlay URL to customize its behavior. Every query
+	parameter is optional.
+</p>
 
-<blockquote><strong>Not all of these APIs are currently supported by the URL builder on the homepage.</strong> If you'd like to add support for these APIs in the URL builder, we'll happily accept <a href="https://github.com/BenDMyers/showmy.chat">your contributions on GitHub</a>!</blockquote>
+<blockquote>
+	<strong
+		>Not all of these APIs are currently supported by the URL builder on the
+		homepage.</strong
+	>
+	If you'd like to add support for these APIs in the URL builder, we'll happily
+	accept
+	<a href="https://github.com/BenDMyers/showmy.chat"
+		>your contributions on GitHub</a
+	>!
+</blockquote>
```

Comment:
This breaking apart onto new lines seems a bit much in this case, and inconsistent in ways I'm not sure I expected. For instance, it seems weird to me that `<strong` and its `>` are on separate lines (the same goes for the closing `</strong` tag, the `<a>`, and the `</a>`. Also, `strong` and `a` are inline elements, and I'm not sure I would expect inline elements like this to be required to be broken up onto multiple lines.

Do we have any levers we can pull to tighten up some of this spacing?

Reply:
I suspect this is because of https://prettier.io/docs/en/options.html#html-whitespace-sensitivity.
More deeply explained in: https://prettier.io/blog/2018/11/07/1.15.0.html#whitespace-sensitive-formatting

The TL;DR is Prettier is conservative by default and formats code in a way that won't change functionality.
Adding a newline before/after the brackets here might change functionality, so it opts for the defense approach to formatting.

That being said, I manually added a newline and now it formats as expected, and upon visual inspection, the functionality seems to have stayed the same.